### DISCUSSION TITLE
Improve 'Pawukon Bali' Calendar Plug-in

### DISCUSSION THREAD

**Discussion by andi-pangerang** [Stellarium/stellarium#2572] [Ideas]
When Calendar plug-in introduced, I'm as Indonesian proud w/ **_Pawukon Bali_** is also included into the plug-in. But, for the developer who develope _**Pawukon Bali**_ plug-in, I hope **you could add some features** such as '_Wuku_' (30 name of Weeks), _Pratiti_, _Lintang_, _Ekajalaresi_, _Pararasan_ and _Pancasuda_, so we could enjoy these features for the next update. Thank you!

**Comment by gzotti** [comment]
Hi!
I am happy if we have actual users of these (for Europeans) exceptional calendar systems, and hope I did not make any errors.  

Unfortunately, the source of my wisdom around calendrical diversity, the book "Calendrical Calculations", does not provide the information you are asking for. I could only include it if you can provide compatible algorithms to compute these features, or you could extend the Balinese calendar yourself.
